auth_jwt/models/ir_http.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+import re
+
+from odoo import SUPERUSER_ID, api, models, registry as registry_get
+from odoo.http import request
+
+from ..exceptions import (
+    UnauthorizedMalformedAuthorizationHeader,
+    UnauthorizedMissingAuthorizationHeader,
+    UnauthorizedSessionMismatch,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+AUTHORIZATION_RE = re.compile(r"^Bearer ([^ ]+)$")
+
+
+class IrHttpJwt(models.AbstractModel):
+
+    _inherit = "ir.http"
+
+    @classmethod
+    def _authenticate(cls, endpoint):
+        """Protect the _authenticate method.
+
+        This is to ensure that the _authenticate method is called
+        in the correct conditions to invoke _auth_method_jwt below.
+        When migrating, review this method carefully by reading the original
```

Comment:
is this checked for v15?

Reply:
Compared source code of `_authenticate` between v14 and v15. Code is the same.
Manual check didn't give much as I'm not confident about all details of usage of this module.